workflow/commons/src/main/java/com/datastax/oss/dsbulk/workflow/commons/log/LogManager.java

```diff
@@ -692,8 +694,9 @@ private void appendStatement(Result result, String logFileName, boolean appendNe
     LogManagerUtils.printAndMaybeAddNewLine(format, writer);
     if (result instanceof WriteResult) {
       WriteResult writeResult = (WriteResult) result;
-      if (!writeResult.wasApplied()) {
-        writer.println("Failed writes: ");
+      // In a condition update could be applied, print the failed ones
+      if (writeResult.isSuccess() && !writeResult.wasApplied()) {
+        writer.println("Failed conditional updates: ");
```

Comment:
why changing this message?

Reply:
I figured that the previous one wasn't clear about the fact that this only applies for conditional statements.

But thanks for asking, I just realized that there are 3 typos in the comment above :-D